src/lib/Behat/PageElement/UpdateForm.php

```diff
@@ -61,7 +75,8 @@ public function fillRichtextWithValue(string $value): void
         $summaryField->setValue($value);
     }
 
-    public function getContainerByHeader(string $containerName): NodeElement
+    /** Returns NodeElement that contains all fields for specified content type field type */
```

Comment:
You can add PHPDoc @returns tag (I think Storm should suggest that with alt-enter magic)